src/modules/ekf2/EKF/aid_sources/gnss/gps_control.cpp

```diff
@@ -174,12 +174,13 @@ void Ekf::controlGpsFusion(const imuSample &imu_delayed)
 	}
 }
 
-void Ekf::updateGnssVel(const gnssSample &gnss_sample, estimator_aid_source3d_s &aid_src)
+void Ekf::updateGnssVel(const imuSample &imu_sample, const gnssSample &gnss_sample, estimator_aid_source3d_s &aid_src)
 {
 	// correct velocity for offset relative to IMU
 	const Vector3f pos_offset_body = _params.gps_pos_body - _params.imu_pos_body;
 
-	const Vector3f vel_offset_body = _ang_rate_delayed_raw % pos_offset_body;
+	const Vector3f angular_velocity = imu_sample.delta_ang / imu_sample.delta_ang_dt - _state.gyro_bias;
```

Comment:
can we really assume that `delta_ang_dt` is strictly > 0 or should we use `imu_delayed.delta_ang / math::max(imu_delayed.delta_ang_dt, FLT_EPSILON)` as in `predictCovariance`?

Reply:
I'd like to do everything we can across the drivers themselves, IMU integration, and ImuDownSampler to ensure it, but even with all that it still doesn't hurt to keep a safe constraint.

https://github.com/PX4/PX4-Autopilot/blob/77709c29487c5e8788ce7b28dd969e08b3bd94bf/src/modules/ekf2/EKF/imu_down_sampler/imu_down_sampler.cpp#L44-L49

If anything the limit could be something much more realistic like 1 millisecond (10x smaller than we'd expect) rather than FLT_EPSILON.

I'll make the change consistently everywhere.